tests/django_restframework_gis_tests/urls.py

```diff
@@ -93,4 +93,8 @@
         r'^filters/within_degrees_of_point$',
         views.geojson_location_within_degrees_of_point_list,
         name='api_geojson_location_list_within_degrees_of_point_filter'),
+    url(
```

Comment:
this can be moved to path() or re_path()

Reply:
Yeah, a `path()` would make way more sense, but I think it would make more sense to make a change in another PR changing all those `url` to `path`

Reply:
> Yeah, a `path()` would make way more sense, but I think it would make more sense to make a change in another PR changing all those `url` to `path`

correct, it's not worth to do it in this patch.

Reply:
ok cool